css/templates/lune/components/_line-after.css

```diff
@@ -1,3 +1,13 @@
 /* //////////////////////////////////////////////////////
   LINE AFTER COMPONENT
-///////////////////////////////////////////////////////*/
\ No newline at end of file
+///////////////////////////////////////////////////////*/
+
+/* To be used in conjunction with bass css 'flex items-center' classes */
+.lune-line-after::after {
+  display: inline-block;
+  flex-grow: 1;
+  height: 2px;
+  margin-left: 0.75rem;
+  background: currentColor;
+  content: '';
+}
```

Comment:
This component creates a line after a heading and grows to fill the empty space.